plugins/kubernetes/app/views/samson_kubernetes/_build_show.erb

```diff
@@ -0,0 +1,4 @@
+<div class="form-group">
+  <%= check_box_tag :kubernetes_job%>
+  <%= form.label :kubernetes_job, "Use Remote Kubernetes Job", class: "control-label" %>
```

Comment:
the new build form gets checkbox ... rebuild does not really need it ... can always make a new build if someone wants to build without remote